force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -40,17 +40,23 @@ public with sharing class ClassesObjectsMethods {
      */
     public static List<String> practiceStringMethods() {
         // Initialize a String with 'Hello, Salesforce!'
-        
-        // Get the length of the String        
+        String str = 'Hello, Salesforce!';
+
+        // Get the length of the String
+        Integer intLen = str.length();        
         // Get a substring of the String of the first 5 characters
+        String strSub = str.substring(0, 5);
         // Convert the String to lower case
-        
+        String strLow = str.toLowerCase();
         // Return the list with the string data
+        List<String> hello = new List<String>();
         // In the first position add the string length
+        hello.add(String.valueOf(intLen));
         // In the second position add the substring
+        hello.add(strSub);
         // In the third position add the string in lower case
-
-        return null; // Replace null with the variable you used to store the result
+        hello.add(strLow);
+        return hello; // Replace null with the variable you used to store the result
```

Comment:
Good work. I  would add my the results directly to my String List rather than creating a bunch of variables. 

```
 public static List<String> practiceStringMethods() {
        List<String> strings = new List<String>();
        String helloSalesforce = 'Hello, Salesforce!';
        strings.add(String.valueOf(helloSalesforce.length()));
        strings.add(helloSalesforce.substring(0, 5));
        strings.add(helloSalesforce.toLowerCase());
        return strings;
    }
    ```